### PR TITLE
Add/webinars banner

### PR DESCRIPTION
**Pull request by wpalani**
## Proposed changes
Add a new section in the App's Home page to display the next Bluehost webinar.

The data of the webinars will be hosted and manually maintained at:
`https://cdn.hiive.space/resources/bluehost-webinars.json`

This PR introduces a new component, `WebinarsBanner` with the following functionality:
* The component will attempt to fetch the webinars data from the Hiive CDN.
* If the file is fetched successfully, the component will verify the required shape of the data.
* When the data is valid, the component will filter the webinars to only include the next webinar.
* The component will then render the next webinar.
* If any of the above fails, the component will not render.

This PR aims to address this internal ticket: https://jira.newfold.com/projects/PRESS1/issues/PRESS1-176

#### Demo
webinar-ad

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by wpscholar** [comment]
@wpalani https://cdn.hiive.space/resources/bluehost-webinars.json doesn't seem to be publicly available.

**Comment by wpalani** [comment]
> @wpalani https://cdn.hiive.space/resources/bluehost-webinars.json doesn't seem to be publicly available.

That's because there's no file there. @chrisdavidmiles will be adding the json file when he gets the content from Marketing.

But if the file is there, it would be accessible. See: https://cdn.hiive.space/resources/bluehost-webinars-example.json

File: src/app/components/webinars-banner/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+			setWebinars(data.items);
+		}
+	}
+
+	useEffect(() => {
+		setNextWebinar(getNextWebinar());
+	}, [webinars]);
+
+	const getNextWebinar = () => {
+		const currentDate = new Date();
```
Comment (circlecube) [inline src/app/components/webinars-banner/index.js:53]: Should we use a timezone offset here? This returns UTC I believe and the Webinars are in EST it looks like. However, it looks like we're just using a date check and not a datetime? Do we still want to display a webinar if it's only a few hours away?

Comment (chrisdavidmiles) [reply src/app/components/webinars-banner/index.js:53]: I'll make sure to convert to utc then. I can remember that.

**Comment by circlecube** [comment]
@chrisdavidmiles, once we have that final webinars.json file uploaded to the CDN we can get this merged.

**Pull request merged**